ledger/participant-integration-api/src/main/resources/db/migration/postgres-appendonly/V106__add_rejection_status_proto_column.sql

```diff
@@ -0,0 +1,16 @@
+--  Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+--  SPDX-License-Identifier: Apache-2.0
+
+-- Store the completion status as a serialized Protocol Buffers message of type `google.rpc.Status`.
+
+-- The `rejection_status` contains a Protocol-Buffers-serialized message of type
+-- `google.rpc.Status`, containing the code, message, and further details (decided by the ledger
+-- driver). We do not migrate the `rejection_status_code` and `rejection_status_message` columns,
+-- and so they may contain old data, which means we need to keep them.
+
+ALTER TABLE participant_command_completions
+    RENAME COLUMN status_code TO rejection_status_code;
+ALTER TABLE participant_command_completions
+    RENAME COLUMN status_message TO rejection_status_message;
+ALTER TABLE participant_command_completions
+    ADD COLUMN rejection_status BYTEA;
```

Comment:
Do we have any HA deployments that might be affected by this?

Reply:
The participant is not highly available; HA is managed by running multiple participants, which do not share an index.

Reply:
not sure about the question but that is not entirely true. with canton and HKEX we will have participant running in HA, sharing a replicated database, in a non-coordinated lock based fashion. PoC code with feature flag already landed. It is activa passive setup (so no load balancing), with ensuring only one indexer is active at the time.

Reply:
That's info I wasn't aware of. Thanks!